Translation corrections (#121)
07th-mod/higurashi-console-arcs: Update/some_010.txt

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#fda762>千紗登</color>", NULL, "<color=#fda762>Chisato</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 37, "ps3/s04/37/310300596", 256, TRUE);
	OutputLine(NULL, "「……私と珠子はね、ちょっと期待していたんだよ。",

> 			NULL, "\"......Tamako and I, we were kind of concerned.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"......Tamako and I were kind of expecting, you know.", Line_WaitForInput);
```

	ModPlayVoiceLS(4, 37, "ps3/s04/37/310300597", 256, TRUE);
	OutputLine(NULL, "あんたがひどいヤツだってことを」",

> 			NULL, " That you were that terrible person.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " That you were the one who did something terrible.\"", Line_Normal);
```

//r開口一番に予想外の言葉を聞かされて、暁は一瞬目を丸くする。しかし、千紗登は暗い表情のまま、話を続けた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "開口一番に予想外の言葉を聞かされて、暁は一瞬目を丸くする。しかし、千紗登は暗い表情のまま、話を続けた。",

> 			NULL, "Hearing those unexpected words, Akira immediately opened his eyes wide. However, Chisato maintained her dark expression and continued speaking.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Akira's eyes widened from the unexpected words. But Chisato maintained her solemn expression and continued.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#fda762>千紗登</color>", NULL, "<color=#fda762>Chisato</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 37, "ps3/s04/37/310300600", 256, TRUE);
	OutputLine(NULL, "「もちろんそうなったら私たちが全力でかばうつもりだったけど、",

> 			NULL, "\"Of course, we were going to do everything in our power to protect her if necessary,", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"Of course, we were going to do everything in our power to cover for her if necessary,", Line_WaitForInput);
```

//rまして今回、…夏美のアルバイト先で殺人事件が起こったのだ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "まして今回、…夏美のアルバイト先で殺人事件が起こったのだ。",

> 			NULL, "Only this time... a murder occurred at Natsumi's part-time job.", Line_Normal);

Reviewer: ```suggestion
			NULL, "But now... a murder occurred at Natsumi's part-time job.", Line_Normal);
```

	ClearMessage();

//rだからもしも、夏美のアルバイトのことが皆に知られてしまうと、クラスの噂好きたちは事件の詳細を知りたがって、彼女に対して無遠慮で無神経な質問を投げかけてくるだろう。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "だからもしも、夏美のアルバイトのことが皆に知られてしまうと、クラスの噂好きたちは事件の詳細を知りたがって、彼女に対して無遠慮で無神経な質問を投げかけてくるだろう。",

> 			NULL, "So, if everyone found out about Natsumi's part-time job, the classmates who love gossip would rudely ask her insensitive questions, hoping to hear the details of the incident.", Line_Normal);

Reviewer: ```suggestion
			NULL, "So if everyone found out about Natsumi's part-time job, her gossip-hungry classmates would ask her all sorts of rude and insensitive questions, hoping to hear more about the incident.", Line_Normal);
```

	ClearMessage();

//rそうなれば、夏美の精神的なダメージが悪化することにもなりかねない。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そうなれば、夏美の精神的なダメージが悪化することにもなりかねない。",

> 			NULL, "If that happened, Natsumi's mental health might deteriorate even further.", Line_Normal);

Reviewer: ```suggestion
			NULL, "If that happened, her mental stability might deteriorate even further.", Line_Normal);
```

	ClearMessage();

//rだから千紗登も珠子も、親しい友達にすらこのことをずっと、話さなかった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "だから千紗登も珠子も、親しい友達にすらこのことをずっと、話さなかった。",

> 			NULL, "So even her closest friends, Chisato and Tamako, didn't want to broach the subject for a long time.", Line_Normal);

Reviewer: ```suggestion
			NULL, "That was why even Chisato and Tamako, her closest friends, didn't want to broach the subject for a long time.", Line_Normal);
```

//r愕然とした思いで、絶句する。他人の悪口を、後ろめたさもなく言いふらす夏美。…悪意と軽蔑を表情に浮かべる彼女の姿が、暁には想像も出来なかった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "愕然とした思いで、絶句する。他人の悪口を、後ろめたさもなく言いふらす夏美。…悪意と軽蔑を表情に浮かべる彼女の姿が、暁には想像も出来なかった。",

> 			NULL, "Akira was stunned speechless. Natsumi, of all people, speaking ill of others without guilt? He couldn't imaginee seeing her with malice and contempt on her face.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Akira was stunned speechless. Natsumi, of all people, speaking ill of others without any remorse? He couldn't imagine seeing her with malice and contempt on her face.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#fda762>千紗登</color>", NULL, "<color=#fda762>Chisato</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 37, "ps3/s04/37/310300619", 256, TRUE);
	OutputLine(NULL, "「そりゃ、私もそう思ってるし、口にも出してたよ？",

> 			NULL, "\"Well, I think so too. and I told them the same.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"Well, I think so too. And I told them the same.", Line_WaitForInput);
```

			NULL, "\"...I don't believe it yet.", Line_WaitForInput);
	ModPlayVoiceLS(4, 37, "ps3/s04/37/310300623", 256, TRUE);
	OutputLine(NULL, "その子、夏美のことあまり好きじゃないみたいだったから、根も葉もない嘘を私たちに吹き込んでるんだって、思ってる。",

> 			NULL, " That student doesn't seem to like Natsumi very much, so I think it was a groundless lie.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " That student doesn't seem to like Natsumi very much, so I think it was a baseless lie.", Line_WaitForInput);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#c8647c>暁</color>", NULL, "<color=#c8647c>Akira</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 39, "ps3/s04/39/310200117", 256, TRUE);
	OutputLine(NULL, "「あ、ああ…」",
			NULL, "\"Y-Yeah...\"", Line_Normal);
	ClearMessage();

//r愚にもつかない話だった。雛見沢の出身者はオヤシロさまの祟りを受けて異常行動を起こし、人を襲ったり自ら命を絶ったりする、…などオカルト的な噂が、暁のクラスでも面白おかしく語られている。
	if (GetGlobalFlag(GADVMode)) {
		OutputLineAll("", NULL, Line_ContinueAfterTyping);
		OutputLine(NULL, "愚にもつかない話だった。雛見沢の出身者はオヤシロさまの祟りを受けて異常行動を起こし、人を襲ったり自ら命を絶ったりする、…などオカルト的な噂が、暁のクラスでも面白おかしく語られている。",

> 				NULL, "<size=-2>It was a ridiculous story. Rumors straight out of a horror movie, such as natives of Hinamizawa acting abnormal as a result of Oyashiro-sama's curse and attacking others or killing themselves, were being shared even in Akira's class.", Line_Normal);

Reviewer: ```suggestion
				NULL, "<size=-2>It was a ridiculous story. Rumors straight out of a horror movie were being shared even in Akira's class. Things like natives of Hinamizawa acting abnormal and attacking others or killing themselves as a result of Oyashiro-sama's curse.", Line_Normal);
```

	} else {
		OutputLine(NULL, "愚にもつかない話だった。雛見沢の出身者はオヤシロさまの祟りを受けて異常行動を起こし、人を襲ったり自ら命を絶ったりする、…などオカルト的な噂が、暁のクラスでも面白おかしく語られている。",

> 				NULL, "It was a ridiculous story. Rumors straight out of a horror movie, such as natives of Hinamizawa acting abnormal as a result of Oyashiro-sama's curse and attacking others or killing themselves, were being shared even in Akira's class.", Line_Normal);

Reviewer: ```suggestion
				NULL, "It was a ridiculous story. Rumors straight out of a horror movie were being shared even in Akira's class. Things like natives of Hinamizawa acting abnormal and attacking others or killing themselves as a result of Oyashiro-sama's curse.", Line_Normal);
```

	}
	ClearMessage();

//rとはいえ別に興味もなかったし、いずれ忘れる与太話だろう、と思って聞き流していたのだが…。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "とはいえ別に興味もなかったし、いずれ忘れる与太話だろう、と思って聞き流していたのだが…。",

> 			NULL, "However, he was not interested in it and ignored it, figuring it was just idle gossip that'd soon be forgotten...", Line_Normal);

Reviewer: ```suggestion
			NULL, "However, he had no interested in that and ignored it, figuring it was just idle gossip that'd soon be forgotten...", Line_Normal);
```
